nixos/modules/services/misc/gitea.nix

```diff
@@ -542,7 +542,7 @@ in
         PrivateMounts = true;
         # System Call Filtering
         SystemCallArchitectures = "native";
-        SystemCallFilter = "~@clock @cpu-emulation @debug @keyring @memlock @module @mount @obsolete @raw-io @reboot @resources @setuid @swap";
+        SystemCallFilter = "~@clock @cpu-emulation @debug @keyring @memlock @module @mount @obsolete @raw-io @reboot @resources @swap";
```

Comment:
This should probably be made conditional on whether `MAILER = "sendmail"` because when using smtp there is no need to allow `setuid`.

Reply:
@joepie91 are you able to take care of this?

Reply:
Missed the comment, sorry. Unfortunately not able to handle this right now - I'll likely be stuck in construction noise for the next couple of weeks...